### PR TITLE
Build release series PRs against dunfell, not master.

### PR DESCRIPTION
**Pull request by kacf**
This enables two things: Release series do not need to deal with
instability or incompatibility in meta-mender/master. We still test
master services with meta-mender/master, allowing easy development of
new features.

Signed-off-by: Kristian Amlie <kristian.amlie@northern.tech>

### CODE REVIEW

File: menderqa_pipeline.go [hunk 1]
```diff
@@ -276,10 +278,18 @@ func getBuildParameters(log *logrus.Entry, conf *config, build *buildOptions) ([
 
 	// set the poky branch equal to the meta-mender base branch, unless it
 	// is master, in which case we rely on the default.
```
Comment (lluiscampos) [inline menderqa_pipeline.go:280]: This comment is a bit off now, it only applies for PRs on `meta-mender` repository.

**Review by lluiscampos — APPROVED** [review]

File: menderqa_pipeline.go (hunk 1 above)
Comment (kacf) [reply menderqa_pipeline.go]: Indeed. Your comment also made me realize I had done it wrong. I was only setting poky, but I need to set meta-mender too. Check out the new one.

**Comment by lluiscampos** [comment]
The rework looks good. Good catch with setting `meta-mender` too. And great comment!

The test fails because we removed reporting service from the releasable components. See https://github.com/mendersoftware/integration/commit/6afe4ac66d9efe49785ed4e15958e1df6963b309 But instead of fixing the tests (and then amending), let's just revert that commit in master. (I am following up)

**Comment by lluiscampos** [comment]
> The test fails because we removed reporting service from the releasable components. See [mendersoftware/integration@6afe4ac](https://github.com/mendersoftware/integration/commit/6afe4ac66d9efe49785ed4e15958e1df6963b309) But instead of fixing the tests (and then amending), let's just revert that commit in master. (I am following up)

@kacf See https://github.com/mendersoftware/integration/pull/1594

**Comment by kacf** [comment]
> @kacf See [mendersoftware/integration#1594](https://github.com/mendersoftware/integration/pull/1594)

Thanks!

**Comment by kacf** [comment]
Updated with acceptance test.

File: tests/tests/payloads/issue_comment_minor_series.json [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+    "url": "https://api.github.com/repos/mendersoftware/mender/issues/865",
+    "repository_url": "https://api.github.com/repos/mendersoftware/mender",
+    "labels_url": "https://api.github.com/repos/mendersoftware/mender/issues/865/labels{/name}",
+    "comments_url": "https://api.github.com/repos/mendersoftware/mender/issues/865/comments",
+    "events_url": "https://api.github.com/repos/mendersoftware/mender/issues/865/events",
+    "html_url": "https://github.com/mendersoftware/mender/pull/865",
+    "id": 1014835255,
+    "node_id": "PR_kwDOAtVyDs4sn41p",
+    "number": 865,
+    "title": "Dummy PR, ignore.",
```
Comment (lluiscampos) [inline tests/tests/payloads/issue_comment_minor_series.json:13]: Good job :+1:

**Review by lluiscampos — APPROVED** [review]
Thanks for adding the test

File: tests/tests/payloads/issue_comment_minor_series.json (hunk 2 above)
Comment (kacf) [reply tests/tests/payloads/issue_comment_minor_series.json:13]: > Good job +1

Your tips were invaluable!

**Pull request merged**